src/lib/components/table/QTable.svelte

```diff
@@ -156,14 +164,14 @@
       size="sm"
       design="flat"
       disabled={page === 1}
-      on:click={() => (page = page - 1)}
+      onclick={() => (page = page - 1)}
     />
     <QBtn
       icon="chevron_right"
       size="sm"
       design="flat"
       disabled={page * rowsPerPage >= rows.length}
```

Comment:
If we have this, why do we have the effect on like 35? Also, why is it `page - 1` on line 35 and `page` here?

Reply:
WDYM? This one just decreases the page number by subtracting `1`.